src/Plugin/WebformElement/CvrCityElement.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Drupal\os2forms_cvr_lookup\Plugin\WebformElement;
+
+use Drupal\webform\Plugin\WebformElement\TextField;
+
+/**
+ * @WebformElement(
+ *   id = "cvr_city_element",
+ *   label = "CVR City Element",
+ *   description = "CVR City Element description",
```

Comment:
This is true, but a very generic description.

Reply:
Fixed in 636b1b1